fabric-entity-events-v1/src/main/java/net/fabricmc/fabric/api/entity/event/v1/ServerPlayerEvents.java

```diff
@@ -45,6 +47,31 @@
 		}
 	});
 
+	/**
+	 * An event that is called when a player takes fatal damage.
+	 *
+	 * <p>Mods can cancel this to keep the player alive.
+	 *
+	 * <p>Vanilla checks for player health <= 0 each frame (with {@link LivingEntity#isDead()}), and kills if true -
+	 * so the player will die next frame if this is cancelled. It's assumed that the listener will do something to
```

Comment:
```suggestion
	 * so the player will die next tick if this is not cancelled. It's assumed that the listener will do something to
```

Reply:
Should be 'so the player will still die next tick it this event is cancelled' - I'll commit it when I'm at my computer.